index.html

```diff
@@ -7,19 +7,45 @@
 
 	<body>
 		<header>
-			<p>This will be the nav</p>
+			<a href="index.html" class="logo">Skinclusive</a>
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
-			<!-- This box contains the large image and text first seen on page-->
+			<!-- This box contains the large image and text first seen on page -->
 			<div class="box1">
-				<p>This will be an image</p>
 				<h1>Skinclusive<br>inclusive of your skin.</h1>
 			</div>
 			
 			<!-- This box contains the main text on the screen when you scroll down-->
 			<div class="box2">
-				<p>Text will go here</p>
+				<p>Let’s face it. Skin care can be a tricky thing to get right. But the truth is, it’s a completely different 

```

Comment:
In my final layout, I began adding some of the actual text which will be displayed on the home page rather than adding dummy text to give me a clearer idea of how my final website will begin to look like.